UI/Pages/Shared/_Layout.cshtml

```diff
@@ -102,7 +102,7 @@ n&&j.setAttribute('nonce',n.nonce||n.getAttribute('nonce'));f.parentNode.insertB
     {
         <!-- Google Tag Manager (noscript) -->
         <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=@containerId"
-height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+height="0" width="0" class="govuk-visually-hidden"></iframe></noscript>
```

Comment:
Is there a reason this is using `govuk-visually-hidden` rather than `govuk-!-display-none`?

Reply:
No reason other than stupidity :-)

Updating that now, good spot.